scripts/bert/finetune_classifier.py

```diff
@@ -424,15 +431,61 @@ def train(metric):
         model.save_parameters(params_saved)
         logging.info('params saved in : {0}'.format(params_saved))
         toc = time.time()
-        logging.info('Time cost=%.1fs', toc - tic)
+        logging.info('Time cost=%.2fs', toc - tic)
         tic = toc
 
 
+def inference(metric):
+    """Inference function."""
+
+    logging.info('Now we are doing BERT classification inference on {} !'.format(ctx))
+    model = BERTClassifier(bert, dropout=0.1, num_classes=len(task.get_labels()))
+    model.hybridize()
```

Comment:
Does static_alloc=True improve inference speed?

Reply:
It's slightly better. So I add static_alloc=True here.